stable/traefik/templates/dashboard-ingress.yaml

```diff
@@ -1,4 +1,4 @@
-{{- if .Values.dashboard.enabled }}
+{{- if and .Values.dashboard.enabled .Values.dashboard.ingressEnabled }}
```

Comment:
Not sure I like this logic.

If you set `dashboard.enabled` to `true`, and `dashboard.ingressEnabled` to false, this still executes.

Also, we want the ingress to be the default option, so having to set two flags to enable the default is not ideal.

Reply:
I wanted to make it possible to disable dashboard ingress but leaving ability to access dashboard via port forwarding. In a way it is currently done - it is all or none.

Reply:
I wanted to make it possible to disable dashboard ingress but leaving ability to access dashboard via port forwarding. In a way it is currently done - it is all or none. I can take it out from this PR, but I think it is common to be able to enable ingress for some services only if needed.